apps/masterbots.ai/components/layout/profile/hero.tsx

```
@@ -0,0 +1,42 @@
+  // get user info
+   async function UserInfoInit(){
+    const {user, error} =  await  getuserInfo(slug as string)
+    if (error) {
+      console.log(error)
+    }
+    setUser(user)
+    
+  }
+
+
+  useEffect(() => {
```

Biome

[error] This hook does not specify all of its dependencies: UserInfoInit

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)